etcdserver/server.go

```diff
@@ -192,6 +192,11 @@ type EtcdServer struct {
 	// when there is no error
 	readNotifier *notifier
 
+	// cmu protects channels below
```

Comment:
is this related to the corruption detection change?

Reply:
Yes, since channel initialization now happens after `v3rpc.monitorLeader` started.

1. `servePeers`
2. `v3rpc.newStreamInterceptor` with `v3rpc.monitorLeader`
3. `e.Server.Start` that initializes the channels